vocabulary-care-pathways/api/src/main/resources/care-pathways/conf/solrcore.properties

```diff
@@ -0,0 +1 @@
+phenotips.version=${phenotips.version}
```

Comment:
Huh, we're going to have a problem if we change the version of this vocabulary but not the version of PhenoTips, the change won't be picked up... That's a problem for upstream.